awot/graph/sonde.py

```diff
@@ -326,10 +338,10 @@ def plot_wind_barbs(data, **kwargs):
     Need to make this a function of the number of
     observations per second/vertical velocity

```

Comment:
Not sure either. Dropsondes have really high wind resolution so it crowds the plot with barbs. Wanted to see what you wanted to do with that.